ConvenientChests/CategorizeChests/Interface/WidgetHost.cs

```diff
@@ -5,28 +5,29 @@
 using StardewModdingAPI.Events;
 using StardewValley;
 
-namespace ConvenientChests.CategorizeChests.Interface
-{
-    class WidgetHost : BaseOverlay
-    {
+namespace ConvenientChests.CategorizeChests.Interface {
+    class WidgetHost : BaseOverlay {
         public readonly Widget RootWidget;
         public readonly ITooltipManager TooltipManager;
 
-        public WidgetHost(IModEvents events, IInputHelper input)
-            : base(events, input)
-        {
-            RootWidget = new Widget() {Width = Game1.viewport.Width, Height = Game1.viewport.Height};
+        public WidgetHost(IModEvents events, IInputHelper input, IReflectionHelper reflection)
+            : base(events, input, reflection, assumeUiMode: true) {
```

Comment:
`assumeUiMode: true` fixes the mouse click coordinates.  
The rest is just updating the BaseOverlay class to the newest version (thanks again @Pathoschild 👍) and me commiting my line-endings 👎